packages/data-stores/src/mapped-types.ts

```diff
@@ -17,8 +22,8 @@
  *
  * @template S Selector map, usually from `import * as selectors from './my-store/selectors';`
  */
-export type SelectFromMap< S extends Record< string, ( ...args: any[] ) => any > > = {
-	[ selector in keyof S ]: (
+export type SelectFromMap< S extends object > = {
+	[ selector in FunctionKeys< S > ]: (
 		...args: TailParameters< S[ selector ] >
 	) => ReturnType< S[ selector ] >;
 };
```

Comment:
This is a slight improvement that allowed it to work on objects that may have non-function properties.

Reply:
TIL.